packages/shared/test-suites/RecordingTestMetadata.ts

```diff
@@ -331,8 +331,9 @@ export async function processCypressTestRecording(
 
     const navigationEvents: RecordingTestMetadataV3.NavigationEvent[] = [];
 
-    // Skipped tests won't contain any annotations (include begin/end point)
-    if (result !== "skipped") {
+    // Tests that were not run (skipped and unknown) won't contain any
+    // annotations (include begin/end point)
+    if (result !== "skipped" && result !== "unknown") {
```

Comment:
Actually I'd like to better understand this change.

As long as the annotations array has a test:start and test:end annotation (which it should?) then I think the client could process this data okay. Maybe the thing we should change is actually how we display the test row in `TestRecordingTreeRow` (so we disable "unknown" tests rather than letting you select them?)